src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -230,14 +266,17 @@ struct CORINFO_InstructionSetFlags
 
     }
 
-    uint64_t GetFlagsRaw()
+    uint64_t* GetFlagsRaw()
     {
         return _flags;
     }
 
-    void SetFromFlagsRaw(uint64_t flags)
+    void SetFromFlagsRaw(uint64_t* flags)
     {
-        _flags = flags;
+        for (int i = 0; i < FlagsArrSize; i++)
+        {
+            _flags[i] = flags[i];
+        }
     }
```

Comment:
Do we want/need to still allow getting the raw flags? I think we could just return and enforce usage of `CORINFO_InstructionSetFlags` instead.

It looks like the "raw" APIs are really only used from `class CORJIT_FLAGS` and `setSupportedISAs` which normally abstracts away the underlying `CORINFO_InstructionSetFlags`.

Reply:
Agreed that it makes sense to remove the `SetFromFlagsRaw` but `GetFlagsRaw` is used from SuperPMI(see changes in `src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp`). I might be missing something here but I am not sure how I can work around that.